src/api/spec/controllers/search_controller_spec.rb

```diff
@@ -0,0 +1,107 @@
+require 'rails_helper'
+require 'webmock/rspec'
+
+RSpec.describe SearchController, vcr: true do
+  render_views
+
+  shared_examples 'block anonymous request' do
+    it { expect(response).to have_http_status(:redirect) }
+  end
+
+  shared_examples 'find project' do
+    it { expect(response).to have_http_status(:success) }
+    it { expect(Xmlhash.parse(response.body)).to have_key('project') }
+    it { expect(Xmlhash.parse(response.body)['project']).to have_key('name') }
+    it { expect(Xmlhash.parse(response.body)['project']['name']).to eq(user.home_project.name) }
```

Comment:
Aren't those 3 the same test? If you don't have a key project you will for sure not have `['project']['name']` with some expected content.

Reply:
your suggestion is to keep just the last one?

Reply:
jupp

Reply:
done